SwiftUI_Homework/SwiftUI_Homework/TabView/MainTabView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ContentView.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 16.01.2023.
+//
+
+import SwiftUI
+
+/// Базовый TabView
+struct MainTabView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let blueColor = Color(#colorLiteral(red: 0.2569369539, green: 0.8115655123, blue: 0.9686274529, alpha: 0.4798957875))
```

Comment:
```suggestion
        static let blueColor = Color(#colorLiteral(red: 0.2569369539, green: 0.8115655123, blue: 0.9686274529, alpha: 0.4798957875))
```
луычше вынести в ассеты а здесь просто будет стринга с названием